File: leet_easy_c++/leet_026/leet_026a.cpp
class Solution {
public:
    int removeDuplicates(vector<int>& nums) {
        int ns = nums.size();
        int cur_num;
        if(ns==0) return 0;
        int last = nums[0];
        int itr = 0;
        for(int cur=1; cur< ns; cur++){
            cur_num = nums[cur];
            if(cur_num!=last){
                itr++;
                nums[itr] = cur_num;
                last = cur_num;
            }    
        }
        return ++itr;
    }
};

File: leet_easy_python/leet_026/leet_026a.py
class Solution(object):
    def removeDuplicates(self, nums):
        sz = len(nums)
        if sz==0: return 0
        i = 1
        cnt = 0
        while i<sz:
            if nums[i] != nums[cnt]:
                cnt+=1
                nums[cnt] = nums[i]
            i+=1
        return cnt+1
        
        """
        :type nums: List[int]
        :rtype: int
        """
        